javascript/spread-operator.md

```diff
@@ -0,0 +1,34 @@
+### The Spread Operator
+
+A powerful feature of ES6, the latest standard of JavaScript implementation, is the spread operator, represented by three dots `...`.
+
+This operator is commonly used as a prefix for parameter names or elements in an array. In this way, expressions can be expanded in places where multiple arguments (for function calls) or multiple elements (for array literal notation) are expected.
+
+* *Example 1 - Function calls*   
+The spread syntax enables you to pull a real array out of the function arguments by adding a parameter prefixed by `...`. This is quite handy when an array is passed as an argument (or multiple arguments) to a function, as below:
+```
+function add(a,b,c) {
+	return a+b+c;
+}
+var scores = [10,20,30];
+add(...scores); // returns 60
+```  
+
+* *Example 2 - Array literal notation*  
+When you declare an array using brackets, you may use the spread operator to populate the array with a number of elements as below:
+```
+var shopping = ["milk","bread","eggs"];
+var prepForParty = ["beers","snacks","sodas"];
+var toBuy = ["rice",...shopping,...prepForParty,"salad"];
+console.log(toBuy); // [ "rice", "milk", "bread", "eggs", "beers", "snacks", "sodas", "salad" ]
+```  
+
+The spread operator looks exactly the same as the rest operator, which is used for destructuring arrays and objects. 
```

Comment:
Perhaps you could add a link to the existing README we have (ES6) that includes the spread and the rest operators.